build/docs/theme/class.latte

```diff
@@ -275,7 +275,7 @@
         </div>
         {/if}
 
-        {if $ownMagicMethods}
+        {if $ownMagicMethods && (!$class->parentClass || $class->parentClass->name !== 'Aws\AwsClient')}
```

Comment:
Is there a way to comment in latte scripts? If so, this definitely warrants a comment to describe why this is needed.